tck/src/main/java/org/eclipse/microprofile/openapi/apps/petstore/resource/PetResource.java

```diff
@@ -150,7 +153,7 @@ public void write(OutputStream output) throws IOException {
     public Response deletePet(
             @Parameter(name = "apiKey", description = "authentication key to access this method",
                        schema = @Schema(type = SchemaType.STRING, implementation = String.class, maxLength = 256,
-                                        minLength = 32)) @HeaderParam("api_key") String apiKey,
+                                        minLength = 32)) @HeaderParam("apiKey") String apiKey,
```

Comment:
It's a bit weird that we have `name` set on the `@Parameter` annotation at all, given that the name can be found from the `@HeaderParam`, but I guess if the test does have it set, the two should match.